app/cafe.py

```diff
@@ -0,0 +1,25 @@
+from app import errors
+from datetime import date
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str | None:
```

Comment:
You can do it like this:
`def visit_cafe(self, visitor: dict) -> str | NotVaccinatedError:`